en/editor/project/index.md

```diff
@@ -81,6 +169,66 @@ If you want to replace all the preset options currently used for image asset, yo
 
 ![Layers-camera](./index/layers-camera.png)
 
+## Physics
+
+![Physics](./index/physics-index.png)
+
+This config of physics will effect only when **Physics Module** is enabled in **Engine Module**.
```

Comment:
This config of physics will effect only when **Physics Module** is enabled in **Engine Module**.
-->
This physics configuration will be in effect only when the **Physics Module** is enabled in the **Engine Module**.